lib/board-io.js

```diff
@@ -288,6 +292,71 @@ BoardIO.prototype.i2cReadOnce = function() {
 };
 
 /**
+ * Sends a config request to the board for hardware or sotware serial
+ * with a required opts parameter that must include the serial port ID,
+ * baud rate, and if software serial, the transmit and receive pins
+ * @param {number} delay in microseconds to set for I2C Read
```

Comment:
The param documentation is incorrect. Use this instead: https://github.com/jgautier/firmata/blob/master/lib/firmata.js#L1487-L1492